packages/maskbook/src/extension/options-page/DashboardDialogs/keystore.ts

```diff
@@ -58,9 +67,9 @@ export function exportFromV3Keystore(input: string | V3Keystore, password: strin
         throw new Error('Unsupport key derivation scheme')
     }
     const ciphertext = Buffer.from(json.crypto.ciphertext, 'hex')
-    const mac = keccak256(Buffer.concat([Buffer.from(derivedKey.slice(16, 32)), ciphertext]))
-    console.log(mac.toString('hex'))
-    if (mac.toString('hex') !== json.crypto.mac) {
+    const buf = Buffer.concat([Buffer.from(derivedKey.slice(16, 32)), ciphertext])
+    const mac = Web3Utils.sha3(`0x${buf.toString('hex')}`)
```

Comment:
@guanbinrui is removing web3 from the repo. I don't think add new usage to web3 is a good idea

Reply:
It's OK for this PR. I will replace it with another utility function.